rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -213,6 +220,9 @@ def _precompiled_apple_resource_bundle_impl(ctx):
             #    Foo.bundle directory that contains our real resources
             unprocessed = [
                 (output_bundle_dir.basename, None, depset([output_bundle_dir, output_plist])),
+                # Handles an edge case when using `rules_xcodeproj`
+                # TODO: Link the PR here
```

Comment:
Reminder to put PR link.

Reply:
https://github.com/bazel-ios/rules_ios/pull/747/commits/2eb797e05ba9f2416b6b3a8f7923c7b5d041bebc